enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -699,20 +712,14 @@ still be created to add a namespace to a mesh.)
 
 #### Can we use the Gateway API v1beta1 CRDs?
 
-OSSM 2.3 is based on Istio 1.14, which only supports Gateway API v1alpha2
-whereas OSSM 2.4 is based on Istio 1.16, which supports Gateway API v1beta1.
-Supporting v1beta1 is highly desirable, and OSSM 2.4 brings many other changes
-that are of interest for this enhancement.  If we continue to use OSSM 2.3, we
-will need to continue to install the Gateway API v1alpha2 CRDs.  We need to
-determine whether we will be able to use OSSM 2.4 for this enhancement.
-
-If OSSM 2.4 is not released in time for OpenShift 4.13, we may resort to using a
-pre-release development build of OSSM 2.4 for dev preview.  While not ideal,
-OSSM 2.4 has sufficient advantages to outweigh the disadvantages of using a
-development build in the context of a dev preview.
+_Note: This question was proposed before OSSM 2.3 (Istio 1.14) supported
+v1beta1 CRDs; therefore the original description has been removed._
 
-**Resolution**: TBD.  We will most likely use a pre-release developer build of
-OSSM 2.4.
+**Resolution**: Yes. OSSM 2.3 is based on Istio 1.14, which has recently added
+support for Gateway API v1beta1. OSSM 2.4 is based on Istio 1.16, which also
```

Comment:
Can you add which minor release of OSSM and Istio included what we need.

Reply:
Sure thing, let me track those down.